admin/config.yml

```diff
@@ -303,77 +286,19 @@ collections:
                   fields:
                     - {label: Name, name: text, widget: string}
                     - {label: Link, name: url, widget: string}
-  # Posts Config
```

Comment:
Can you outline the reason for this giant deletion?

Reply:
Or I could look in the commit history. It would be nice if GitHub surfaced the `git blame` information here. :)

Reply:
I've removed the `posts` config section amongst others as 1) it is not needed by connect.linaro.org as the posts are generated via pythons scripts every Connect 2) the path for a collection can only be one folder currently. We need to add/wait for support for path patterns to be added upstream to the netlifycms project (as a workaround, however, we can declare the folder for each Connect as a separate netlifycms collection - or just provide access to edit the *current* connect session posts...) :) 

BR,

Kyle